### PR TITLE
LPD-31824

### PR DESCRIPTION
**Pull request by veroglez**

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

**Comment by veroglez** [comment]
ci:test:page-management

**Comment by veroglez** [comment]
ci:test:page-management

**Comment by veroglez** [comment]
ci:test:page-management

File: modules/apps/layout/layout-api/src/main/java/com/liferay/layout/util/structure/LayoutStructure.java [hunk 1]
```diff
@@ -506,22 +506,42 @@ public LayoutStructureRule deleteLayoutStructureRule(String ruleId) {
 		return layoutStructureRule;
 	}
 
-	public List<LayoutStructureItem> duplicateLayoutStructureItem(
-		String itemId) {
+	public Map<String, List<LayoutStructureItem>> duplicateLayoutStructureItem(
```
Comment (georgel-pop-lr) [inline modules/apps/layout/layout-api/src/main/java/com/liferay/layout/util/structure/LayoutStructure.java:509]: Shouldn't we call this method duplicateLayoutStructureItems now that it affects various items?

File: modules/apps/layout/layout-api/src/main/java/com/liferay/layout/util/structure/LayoutStructure.java [hunk 2]
```diff
@@ -618,44 +638,49 @@ public boolean isPortletMarkedForDeletion(String portletId) {
 	}
 
 	public void markLayoutStructureItemForDeletion(
```
Comment (georgel-pop-lr) [inline modules/apps/layout/layout-api/src/main/java/com/liferay/layout/util/structure/LayoutStructure.java:640]: I think same here, it should be markLayoutStructureItemsForDeletion

**Comment by veroglez** [comment]
ci:forward

**Comment by veroglez** [comment]
ci:test:sf

**Comment by veroglez** [comment]
ci:test:sf

**Comment by veroglez** [comment]
ci:forward

**Pull request closed without merging**